docker-compose-junit-jupiter/src/main/java/com/palantir/docker/compose/DockerComposeExtension.java

```diff
@@ -30,14 +30,24 @@
 public abstract class DockerComposeExtension extends DockerComposeManager
         implements BeforeAllCallback, AfterAllCallback {
 
+    private boolean hasCalledAfterMethod;
```

Comment:
I think this should live in `DockerComposeManager`, not `DockerComposeExtension` that way `DockerComposeRule` gets this behaviour too.